components/metatext_ai_pre_build_ai_models_api/metatext_ai_pre_build_ai_models_api.app.mjs

```diff
@@ -0,0 +1,47 @@
+import { axios } from "@pipedream/platform";
+import constants from "./common/constants.mjs";
+
+export default {
+  type: "app",
+  app: "metatext_ai_pre_build_ai_models_api",
+  propDefinitions: {
+    text: {
+      type: "string",
+      label: "Text",
+      description: "The text to analyze.",
```

Comment:
Since this parameter will not be always "analyzed" the text of this description doesn't not specify the specific usage for each action. 
You could either add a description for this parameter in each Action or use something less specific such as `The text parameter to be used as base for the request.`